README.md

```diff
@@ -20,98 +17,20 @@ To run this CLI application
-
-- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
-- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
-
-This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
-
-- Hardhat => `packages/hardhat/contracts`
-- Foundry => `packages/foundry/contracts`
-
-The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:
-
-- Hardhat => `packages/hardhat/deploy`
-- Foundry => `packages/foundry/script`
-
-4. On a third terminal, start your NextJS app:
-
-```
-yarn start
-```
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
+- They can see clearly which ones are unlocked and which ones are still locked
+- It only allows them to interact with the challenges that are unlocked
+- It shows them their progress/general score on each challenge and total score on each branch of the tree
+- They can download a challenge locally
+- They can deploy and verify their contract with very few steps
+- They can submit the contract address for the challenge and get real-time feedback
+- If they pass the challenge it unlocks the next set of challenges
+- There are 5 - 10 challenges available. We will limit to a few branches at the start
+
+## Future Ambitions
+- Issue onchain attestation for completion of the challenge
+- Front end that shows their progress - phase 1
+- Front end that replaces ETT CLI - phase 2
```

Comment:
I have no idea where the functionality is at right now with the CLI, but if the plan is to fully replace it with front-end, maybe the time is better spent just building the frontend from the start? 

Keep in mind this feedback is biased by my comfort with building frontends, although I would imagine it would be easier to find more devs capable of helping you build a frontend than it would be to find devs with experience building CLI tools

Also please disregard this entire comment if the CLI tool is a breeze for you to setup with all the functionality necessary to meet MVP requirements

Reply:
One thing I should clarify is that when I say "replaces ETT CLI" I don't really mean that ETT CLI will cease to be maintained or promoted. I guess I mean that users could use the web version or the CLI version. 

Really appreciate you being candid about your thoughts. My biggest reasons for starting with the CLI version is that it is the shortest path toward having something in place that meets the markets needs. ETT front ends have been attempted no less than three times and so I am wary of getting stuck in the same places past attempts failed. The thought of doing a CLI instead of a front end was first presented by Carlos, who was in the weeds with past attempts, so I take his suggestion pretty seriously.

I do see the CLI as being relatively simple to complete as well. Definitely easier to complete than the front end.